compilers/concrete-compiler/compiler/tests/end_to_end_tests/end_to_end_test.cc

```diff
@@ -169,7 +170,7 @@ std::string getTestName(EndToEndDesc desc,
                         int testNum) {
   std::ostringstream os;
   os << getOptionsName(options) << "." << desc.description << "." << testNum;
-  return os.str();
+  return std::regex_replace(os.str(), std::regex("-"), "");
```

Comment:
Maybe a comment saying it avoid a feature when - is present in the name

Reply:
I explain that on the commit comment